python-sdk/src/forta_agent/utils.py

```diff
@@ -47,25 +49,33 @@ def create_transaction_event(dict):
     return TransactionEvent(dict)
 
 
-def assert_non_empty_string_in_dict(dict, key):
-    assert_key_in_dict(dict, key)
-    assert isinstance(dict[key], str) and len(
-        dict[key]) > 0, f'{key} must be non-empty string'
+def is_valid_key(func: callable):
+
+    def wrap(dict: dict, key: str or int, *args, **kwargs):
+        if not key in dict:
+            raise KeyError(f'{key} is requierd!')
+        else:
+            return func(dict, key, *args, **kwargs)
+    return wrap
 
 
-def assert_enum_value_in_dict(dict, key, enum):
-    assert_key_in_dict(dict, key)
-    assert isinstance(dict[key], enum), f'{key} must be valid enum value'
+@is_valid_key
+def is_valid_string(dict: dict, key: int or str):
```

Comment:
lets keep the function names the same (i.e. `assert_enum_value_in_dict` and `assert_non_empty_string_in_dict`) as they are used in other places as well